frontend/src/pages/modelServing/screens/projects/ServingRuntimeModal/ManageServingRuntimeModal.tsx

```diff
@@ -199,42 +200,59 @@ const ManageServingRuntimeModal: React.FC<ManageServingRuntimeModalProps> = ({
         </Button>,
       ]}
     >
-      <Form
-        onSubmit={(e) => {
-          e.preventDefault();
-          submit();
-        }}
-      >
-        <Stack hasGutter>
-          <StackItem>
-            <ServingRuntimeReplicaSection data={createData} setData={setCreateData} />
-          </StackItem>
-          <StackItem>
-            <ServingRuntimeSizeSection data={createData} setData={setCreateData} sizes={sizes} />
-          </StackItem>
-          <StackItem>
-            <FormSection title="Model route" titleElement="div">
-              <FormGroup>
-                <Checkbox
-                  label="Make deployed available via an external route"
-                  id="alt-form-checkbox-route"
-                  name="alt-form-checkbox-route"
-                  isChecked={createData.externalRoute}
-                  onChange={(check) => setCreateData('externalRoute', check)}
+      <Stack hasGutter>
+        <StackItem>
+          <Form
+            onSubmit={(e) => {
+              e.preventDefault();
+              submit();
+            }}
+          >
+            <Stack hasGutter>
+              <StackItem>
+                <ServingRuntimeReplicaSection data={createData} setData={setCreateData} />
+              </StackItem>
+              <StackItem>
+                <ServingRuntimeSizeSection
+                  data={createData}
+                  setData={setCreateData}
+                  sizes={sizes}
                 />
-              </FormGroup>
-            </FormSection>
+              </StackItem>
+              <StackItem>
+                <FormSection title="Model route" titleElement="div">
+                  <FormGroup>
+                    <Checkbox
+                      label="Make deployed available via an external route"
+                      id="alt-form-checkbox-route"
+                      name="alt-form-checkbox-route"
+                      isChecked={createData.externalRoute}
+                      onChange={(check) => setCreateData('externalRoute', check)}
+                    />
+                  </FormGroup>
+                </FormSection>
+              </StackItem>
+              <StackItem>
+                <ServingRuntimeTokenSection data={createData} setData={setCreateData} />
+              </StackItem>
+            </Stack>
+          </Form>
+        </StackItem>
+        {error && (
+          <StackItem>
+            <Alert isInline variant="danger" title="Error creating model server">
+              {error.message}
+            </Alert>
           </StackItem>
+        )}
+        {loadErrorServingRuntimeConfig && (
           <StackItem>
-            <ServingRuntimeTokenSection data={createData} setData={setCreateData} />
+            <Alert isInline variant="warning" title="Error fetching model server configuration">
+              {loadErrorServingRuntimeConfig.message}
+            </Alert>
```

Comment:
This is interesting -- why two sets of messages of different intensity? Seems they would both be just _errors_, no?

Reply:
I can ask UX about this, one of the error will prevent the model server to be deployed (i.e there's an error in the creation) and the other one is to display that the default configuration is missing, and I treated that as a warning since the user will be able to deploy the model server.

Reply:
I think two alerts at the bottom feel weird is all I'm getting at. Perhaps show the warning if there is no error? The warning I think will appear first -- then you submit -- then an error -- resubmit would remove the error only after it returns, right? So maybe that's undesirable.

Could you show a mock to UX?

Reply:
Yeah, I'm gonna share a mock to UX, leaving a quick flow here just to clarify things:

* ConfigMap `servingruntime-config` is missing or invalid but there are no errors deploying the model:
   1. A warning alert will appear telling the user a fallback config will be applied.
   2. Model will be deployed once the user click the `deploy` button.

* ConfigMap `servingruntime-config` is valid and there are errors deploying the model:
   1. Modal will open with no alerts.
   2. User will click the `deploy` button.
   3. Model won't deploy, modal won't be dismissed and an error alert will be displayed

* ConfigMap `servingruntime-config` is missing or invalid AND there's an error deploying the model:
   1. A warning alert will appear telling the user a fallback config will be applied.   
   2. User will click the `deploy` button.
   3. Model won't deploy, modal won't be dismissed and an error alert will be displayed alongside the warning alert.

Maybe I can make the warning alert disappear if the error alert is displayed as I think you are suggesting.

Reply:
@vconzola FYI

Reply:
@lucferbux I'm a little confused. Is the mock you're talking about showing UX the one above? That shows a model server config modal, and here it sounds like you're talking about what happens when deploying a model?

Reply:
No no, this is related to when a Model Server is created. There might be some issues/errors when creating the model server, hence we need an alert to display in the modal to warn the user about those errors.